internal/ethapi/api.go

```diff
@@ -2044,6 +2047,24 @@ func runMEVM(ctx context.Context, b Backend, state *state.StateDB, header *types
 		computeResult = result.ReturnData // Or should it be nil maybe in this case?
 	}
 
+	logs := state.GetLogs(common.Hash{}, 0, common.Hash{})
+
+	if len(logs) != 0 {
+		result := suave.ExecResult{
+			Logs: logs,
+		}
+
+		logsEncoded, err := result.EncodeABI()
+		if err != nil {
+			return nil, nil, nil, err
+		}
+
+		// add some magic number and the result (TODO)
+		computeResult = append(computeResult, magicBytes...)
```

Comment:
We should probably only do this if we actually have anything to encode right?

Reply:
There is a check before that only appends if `logs!=0` which means that there is some data to encode here.